docs/install.rst

```diff
@@ -30,22 +30,21 @@ PlasmaPy also depends on the following packages for optional features:
 
 - `h5py <https://www.h5py.org/>`_ — 2.8 or newer
 - `lmfit <https://lmfit.github.io/lmfit-py/>`_ — 1.0.1 or newer
-- `matplotlib <https://matplotlib.org/>`_ — 2.0 or newer
+- `matplotlib`_ — 2.0 or newer
 - `mpmath <https:htt//mpmath.org/>`_ — 1.0 or newer
 
 .. _install-process:
 
 Installing PlasmaPy
 ===================
 
-
 .. _install-pip:
 
 Installation with pip
 ---------------------
 
 To install the most recent release of PlasmaPy on `PyPI`_
-with `pip <https://pip.pypa.io/en/stable/>`_ into an existing Python environment
+with `pip`_ into an existing Python environment
```

Comment:
If we add `pip` to the `intersphinx_mappings` dict, then `` `pip` `` will likely automatically cross-reference.

Reply:
I tried adding `pip` to `intersphinx_mappings` but the link didn't work, so I reverted the change rather than go down a dragon hole.  I'd be okay with this change at some future time.